src/WebRtc/Utils/WebRtcTransfer.php

```diff
@@ -10,11 +10,11 @@
 namespace BandwidthLib\WebRtc\Utils;
 
 class WebRtcTransfer {
-    public static function generateBxml($deviceToken, $sipUri = "sip:sipx.webrtc.bandwidth.com:5060") {
-        return '<?xml version="1.0" encoding="UTF-8"?><Response>' . WebRtcTransfer::generateTransferBxmlVerb($deviceToken, $sipUri) . '</Response>';
+    public static function generateBxml($deviceToken, $voiceCallId, $sipUri = "sip:sipx.webrtc.bandwidth.com:5060") {
```

Comment:
Can you doc the parameters? Here's an example for PHP:

```
/**
 * Add two integers together
 *
 * @param int $a First integer
 * @param int $b Second integer
 * @return int Sum of integers $a and $b
 */
```

Reply:
Added, thank you!